src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -482,7 +489,7 @@ where
                     .await;
 
                 let mut source_upper = reclock_follower
-                    .source_upper_at_frontier(resume_upper.borrow())
+                    .source_upper_at_frontier(initial_resume_upper.borrow())
```

Comment:
nit: Is there a "non-initial" resume upper in this context? Removing the alias from the config destructure seems clearer

Reply:
yeah i can change this back!